Objects/typeobject.c

```diff
@@ -3634,8 +3634,9 @@ check_basicsize_includes_size_and_offsets(PyTypeObject* type)
 }
 
 PyObject *
-PyType_FromMetaclass(PyTypeObject *metaclass, PyObject *module,
-                     PyType_Spec *spec, PyObject *bases_in)
+_PyType_FromMetaclass_impl(
+    PyTypeObject *metaclass, PyObject *module,
+    PyType_Spec *spec, PyObject *bases_in, int _allow_tp_new)
```

Comment:
That's an unusual naming; why the leading underscore in `_allow_tp_new`?

Reply:
It'll go away in a future version. It's even more unstable/internal than the surrounding code.